docs/documentation/models/geometry/wing_tank_model.rst

```diff
@@ -0,0 +1,66 @@
+######################
+Wing tank volume model
+######################
+
+There are several models implemented for the computation of the volume of the tanks inside the wings of the aircraft.
+
+.. contents::
+
+.. _simple-model:
+
+*****************
+Simple tank model
+*****************
+
+The simple tank volume model is based on the assumption that the tank is a parallelepiped.
+
+The base of the tank is computed by making the assumptions that it stretches from the fuselage centerline to the wing span and that it is located between the 1st and 3rd stringer (respectively located at 30% and 60% of the chord).
+
+The height of the tank is computed using an average wing thickness obtained based on the thickness at the tip and root as well as a penalty factor which results in the following formula :
+
+.. math::
+    t_{avg} = 0.7 * \frac{c_{tip} * \left( t/c \right)_{tip} + c_{root} * \left( t/c \right)_{root}}{2}
+
+The volume is then computed with:
+
+.. math::
+    \text{MFV} = 0.3 * S_{w} * t_{avg}
+
+Finally the Maximum Fuel Weight (MFW) is computed based on the density of the fuel (which can be either Jet A1 or AvGas).
+
+This model is the one activated by default.
```

Comment:
Now that I am thinking about it; would it be of interest to give an indication of the model behaviour based on your experience with it? For example, this model usually tends to overestimate the MFW?

Reply:
I don't think I'd be able to say if each models overestimate or underestimate 😅